pom.xml

```diff
@@ -113,4 +132,82 @@
             </plugin>
         </plugins>
     </build>
+    <reporting>
+        <!--https://maven.apache.org/plugins/maven-site-plugin/maven-3.html#Classic_configuration_Maven_2__3-->
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-project-info-reports-plugin</artifactId>
+                <version>2.4</version>
+                <configuration>
+                    <dependencyDetailsEnabled>false</dependencyDetailsEnabled>
+                    <dependencyLocationsEnabled>false</dependencyLocationsEnabled>
+                </configuration>
+                <reportSets>
+                    <reportSet>
+                        <reports>
+                            <report>dependencies</report>
+                            <report>scm</report>
+                        </reports>
+                    </reportSet>
+                </reportSets>
+            </plugin>
+        </plugins>
+    </reporting>
+    <profiles>
+        <profile>
+            <id>jenkins</id>
```

Comment:
Is everything under this profile too expensive to run for normal builds? I ask since I imagine we'd want to eventually turn some of these checks (e.g. jacoco coverage) into blockers for check-in, in which case having them as part of the normal build might be a bit nicer since it catches issues earlier.